src/uucp_break.ml

```diff
@@ -35,6 +35,38 @@ let sentence u = Array.unsafe_get Low.sentence_of_int (Low.sentence u)
 
 let east_asian_width u = Uucp_rmap.get Uucp_break_data.east_asian_width_map u
 
+let err_ctrl_char e =
+  invalid_arg (Printf.sprintf "0x%02X is a control character" e)
+
+let err_not_uchar e =
+  invalid_arg (Printf.sprintf "U+%04X is not a unicode scalar value." e)
+
+let terminal_width_hint =
+  let gc = Uucp_gc.general_category in
+  function
+  (* C0 (without 0x00) + DELETE + C1 is an error. *)
+  |u when 0 < u && u <= 0x1f || 0x7f <= u && u <= 0x9f -> err_ctrl_char u
```

Comment:
> How about -1? A little yuck, but it won't jump into the GC on every call.

Ok. Very yuck, but then it seems a good compromise between robustness in case of a careless client and be nice to the ones who actually do the right thing.